packages/hash/api/src/collab/Instance.ts

```diff
@@ -203,16 +215,107 @@ export class Instance {
         ({ accountId, entityId }) => `${accountId}/${entityId}`,
       );
 
-      const entityVersionTime = new Date(entityVersion.updatedAt).getTime();
-      const entityVersionRef = getEntityRef(entityVersion);
+      /**
+       * This fetches entity references relevant to the provided LinkVersion or AggregationVersion.
+       * Multiple entity refs may be returned, as a LinkVersion has both a source and a destination.
+       * The LinkedEntities for a destination can change if any of its properties are resolved via incoming links.
```

Comment:
I'm not sure I understand this, which properties of an entity are affected by incoming links? Is this related to a upcoming implementation of property formulas?

Reply:
None currently, but we'll need this for things like an Org's members (where we don't handle them), or for 'how many stars has this repo have'. I wrote it to be forward looking to save coming back and updating some 'in the future...' comment.

Reply:
> but we'll need this for things like an Org's members (where we don't handle them)
Just so you're aware, currently we manually maintain the reverse link from an org to a membership, so this wouldn't be needed.

But I understand your reasoning, and think this is fine.